Streaming.css

```diff
@@ -149,14 +227,142 @@ footer {
     bottom: 0;
     background-color: rgb(172, 172, 172);
     left: 0;
+    border-radius: 10px 10px 0px 0px;
 }
-
+main{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    position: absolute;
+    width: 78%;
+    right: 0;
+    height: 70%;
+    bottom: 60px;
+    overflow-y: scroll;
+    margin-top: 50%;
+}
+.video{
+    padding: -20px;
+    position: relative;
+    width: 200px;
+}
+#paw_video{
+    display: none;
+    position: absolute;
+    padding-left: 88px;
+    padding-top: 146%;
+    z-index: 1; 
+    border-radius: 5px;
+    cursor: grab;
+}
+#paw_video:focus .preview{
+    display: flex;
+}
+.preview{
+    width: 200px;
+    height: 131.33px;
+    border: solid 1px black;
+}
+.preview:hover{
+    filter: blur(2px);
+    box-shadow: 0px;
+}
+.live {
+    position: absolute;
+    margin-left: 3px;
+    margin-top: 3px;
+    background-color: rgb(255, 0, 0, 0.78);
+    color: white;
+    border-radius: 5px;
+    animation: live 2s infinite;
+    cursor:grab;
+    z-index: 1;
+}
+nav{
+    display: flex;
+    right: 0;
+}
+.name_v{
```

Comment:
You need fixed width to work text-overflow: ellipsis;

Reply:
Now I know it